### PR TITLE
Add the Loadbalancer tag to ingress tests

### PR DESCRIPTION
**Pull request by jayunit100**
Quick PR to add the Loadbalancer tag to ingress tests

### CODE REVIEW

File: test/e2e/network/ingress.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ const (
 	instanceGroupAnnotation = "ingress.gcp.kubernetes.io/instance-groups"
 )
 
-var _ = SIGDescribe("Loadbalancing: L7", func() {
+var _ = SIGDescribe("Loadbalancing: L7 [[Loadbalancer]", func() {
```
Comment (aojea) [inline test/e2e/network/ingress.go:58]: double `[[` ?

Comment (jayunit100) [reply test/e2e/network/ingress.go:58]: whoops jas fixing

**Comment by robscott** [comment]
This seems like a reasonable change to me. Maybe @spencerhance or @bowei can confirm?

File: test/e2e/network/ingress.go (hunk 1 above)
Comment (jayunit100) [reply test/e2e/network/ingress.go]: fxd

**Comment by jayunit100** [comment]
closing - i think #97472 is a better path forward

**Pull request closed without merging**